flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/config/AutoscalerStandaloneOptions.java

```diff
@@ -59,4 +64,47 @@ private static ConfigOptions.OptionBuilder autoscalerStandaloneConfig(String key
                     .withDeprecatedKeys("flinkClusterPort")
                     .withDescription(
                             "The port of flink cluster when the flink-cluster fetcher is used.");
+
+    public static final ConfigOption<StateStoreType> STATE_STORE_TYPE =
+            autoscalerStandaloneConfig("state-store.type")
+                    .enumType(StateStoreType.class)
+                    .defaultValue(StateStoreType.MEMORY)
+                    .withDescription("The autoscaler state store type.");
+
+    public static final ConfigOption<String> STATE_STORE_JDBC_URL =
+            autoscalerStandaloneConfig("state-store.jdbc.url")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "The jdbc url of jdbc state store when %s has been set to %s, such as: %s.",
+                                            code(STATE_STORE_TYPE.key()),
+                                            code(JDBC.toString()),
+                                            code("jdbc:mysql://localhost:3306/flink_autoscaler"))
+                                    .linebreak()
+                                    .text("This option is required when using JDBC state store.")
+                                    .build());
+
+    public static final ConfigOption<String> STATE_STORE_JDBC_USERNAME =
+            autoscalerStandaloneConfig("state-store.jdbc.username")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "The jdbc username of jdbc state store when %s has been set to %s.",
+                                            code(STATE_STORE_TYPE.key()), code(JDBC.toString()))
+                                    .build());
+
+    public static final ConfigOption<String> STATE_STORE_JDBC_PASSWORD =
+            autoscalerStandaloneConfig("state-store.jdbc.password")
```

Comment:
We need a way to set this from a secret / env. Otherwise this is very hard to be used in production properly